reset.c

```diff
@@ -0,0 +1,888 @@
+/*  reset.c --  removes aligner updates and reference data from input sam/bam/cram file and makes read data raw
+                for new processing
+
+    Copyright (C) 2022 Genome Research Ltd.
+
+    Author: Vasudea Sarma <vasudeva.sarma@sanger.ac.uk>
```

Comment:
- If we're going to line wrap, it's probably best to do so at the same
  column width used elsewhere in the file, which is (usually) 80
  chars.  Eg see the very first comment line for over-long wrapping.

- Typo in author name? Vasudea

Reply:
:-) Done